examples/kubernetes-ingress/scripts/09-cleanup.sh

```diff
@@ -8,6 +8,6 @@ kubectl delete -f "${dir}/../deployments/guestbook/"
 
```

Comment:
Add logs using tests/helpers.bash:log so users can follow what's going on in case they hit an issue while the script runs.